From 8e28f1868aa5542ed8c98a317962af61e301913d Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Thu, 11 Apr 2013 11:07:31 +0200
Subject: [PATCH 017/212] Change qemu to qemu-kvm

We use qemu-kvm as name for RHEL 7. This patch fixes various pathes and files that uses qemu so they use qemu-kvm instead.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>

Conflicts (rebase):
	redhat/qemu-kvm.spec.template - need to handle added files
---
 os-posix.c                    |    2 +-
 redhat/qemu-kvm.spec.template |  128 ++++++++++++++++++++++-------------------
 ui/vnc.c                      |    2 +-
 3 files changed, 70 insertions(+), 62 deletions(-)

diff --git a/os-posix.c b/os-posix.c
index cb2a7f7..452c23f 100644
--- a/os-posix.c
+++ b/os-posix.c
@@ -78,7 +78,7 @@ void os_setup_signal_handling(void)
 /* Find a likely location for support files using the location of the binary.
    For installed binaries this will be "$bindir/../share/qemu".  When
    running from the build tree this will be "$bindir/../pc-bios".  */
-#define SHARE_SUFFIX "/share/qemu"
+#define SHARE_SUFFIX "/share/qemu-kvm"
 #define BUILD_SUFFIX "/pc-bios"
 char *os_find_datadir(void)
 {
diff --git a/ui/vnc.c b/ui/vnc.c
index f8d9b7d..b6ff0ad 100644
--- a/ui/vnc.c
+++ b/ui/vnc.c
@@ -3273,7 +3273,7 @@ void vnc_display_open(DisplayState *ds, const char *display, Error **errp)
     }
 
 #ifdef CONFIG_VNC_SASL
-    if ((saslErr = sasl_server_init(NULL, "qemu")) != SASL_OK) {
+    if ((saslErr = sasl_server_init(NULL, "qemu-kvm")) != SASL_OK) {
         error_setg(errp, "Failed to initialize SASL auth: %s",
                    sasl_errstring(saslErr, NULL, NULL));
         goto fail;
-- 
1.7.1

